API/Controllers/CountryController.cs

```diff
@@ -29,6 +29,51 @@ public IEnumerable<CountryContract> GetAllCountries()
             return countryContracts;
         }
 
+        [HttpGet()]
+        public IEnumerable<string> GetAllCountryNames()
+        {
+            var countries = _countryRepository.GetAllCountries();
+
+            var countryNames = new List<string>();
+
+            foreach(var country in countries)
+            {
+                countryNames.Add(country.Name);
+            }
+
+            return countryNames;
+        }
+
+        [HttpGet()]
+        public IEnumerable<string> GetAllCountryCodes()
```

Comment:
Wouldn't it be better to get all codes form the database in the repository?

Reply:
Removed